Sprint-3/reading-list/script.js

```diff
@@ -1,3 +1,37 @@
+//create a function to display the books in the reading list
+function displayBooks() {
+  const bookListElement = document.getElementById("reading-list");
+  bookListElement.innerHTML = ""; // Clear the existing list
+
+  books.forEach((book) => {
+    // Create a single <li> element
+    const bookElement = document.createElement("li");
+    
+    // Set background color based on alreadyRead status
+    bookElement.style.backgroundColor = book.alreadyRead ? "lightgreen" : "rgb(245, 43, 43)";
+    
+    // Add text content (title & author), adding in paragraph makes the text more readable and styling easier
+    bookElement.innerHTML = `<p>${book.title}</br> by ${book.author.toUpperCase()}</p>`;
+
```

Comment:
Please try  not use innerHTML, if you don't have any other choice, innerHTML is not ideal method  while making DOM manipulation. 
You may refer to this blog 
https://medium.com/@sparklewebhelp/understanding-the-risks-of-using-inner-html-in-web-development-30d4fa67f815